docs/features/role-based-access-control/brief.md

```diff
@@ -0,0 +1,40 @@
+# Role base access control - Brief
+
+This document aims at providing an overview of Role-based Access Control (RBAC from here on) for the Marxan platform. 
+
+In broad strokes, RBAC assumes the existence of a set of roles, which are associated with different actions that they 
+can perform in the platform. Users can then be associated with roles for specific resources, which will grant them the 
+role privileges for that specific resource.
+
+The current implementation is based on the requirements described in the [User requirements document](https://docs.google.com/document/d/16x4wQYTKov0JkO1ue6iQzJ3-RXpdgQ38ygGJWW86Zo0/edit#heading=h.58xbt2es73r2).
+
+## Roles
+
+In the context of the Marxan platform, we will consider the following roles:
+
+* **Admin**: a generic platform administrator, who can perform **any action for any resource** in the platform.
+* **Owner**: represents a user that can perform any management actions for a specific resource of the platform (e.g. 
+a project). This role is typically attributed to the user who created the resource (i.e. they own the resource).
```

Comment:
tiny suggestion: `typically attributed initially`, to clarify that this would be the initial/default case, but more owners may be added later on (or an owner may add another one and remove themselves, for example)